tools/oeutil/host/generate_evidence.cpp

```diff
@@ -1191,6 +1184,7 @@ oe_result_t generate_oe_evidence(
     else
     {
         // Print basic info to console
+        sgx_quote_t* quote = (sgx_quote_t*)evidence;
```

Comment:
Please ensure the evidence format buffer starts with sgx_quote_t.

Reply:
Updated. Ensure evidence format to be `sgx_ecdsa` before dumping.